modules/popup/react/lib/Popup.tsx

```diff
@@ -110,7 +117,13 @@ export default class Popup extends React.Component<PopupProps> {
             />
           </CloseIconContainer>
         )}
-        <Card depth={depth} heading={heading} width={width} padding={padding}>
+        <Card
+          depth={depth}
+          heading={heading}
+          headingId={heading ? this.id : undefined}
```

Comment:
Should we support this as a prop?

Reply:
Hmm. Is there a benefit to exposing this id? We could add it later if we need to.

Reply:
Yeah I guess in my head I always want human readable anchor links for headings but that doesn't make sense in a modal anyways. We can leave out 👍